plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -910,6 +912,255 @@ public void testSettingChangeDataFeedEnabledProperty()
                 .contains("change_data_feed_enabled = true");
     }
 
+    @Test
+    public void testReadingChangesMadeByBasicOperationsOnTableWithCdfEnabled()
+    {
+        String tableName = "test_basic_operations_on_table_with_cdf_enabled_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (page_url VARCHAR, domain VARCHAR, views INTEGER) WITH (change_data_feed_enabled = true)");
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url1', 'domain1', 1), ('url2', 'domain2', 2), ('url3', 'domain3', 3)", 3);
+        assertUpdate("INSERT INTO " + tableName + " VALUES('url4', 'domain4', 4), ('url5', 'domain5', 2), ('url6', 'domain6', 6)", 3);
+
+        assertUpdate("UPDATE " + tableName + " SET page_url = 'url22' WHERE views = 2", 2);
```

Comment:
It would be nice to have an assertion similar to `SHOW COLUMNS FROM xxx` (even if not possible atm on table functions) performed for metadata tables (see `io.trino.plugin.deltalake.TestDeltaLakeSystemTables#testHistoryTable` as reference)

- column name
- column type

Reply:
I am not sure if that would be very useful tbh. I don't really see why someone would use it